pkgs/applications/office/jabref/default.nix

```diff
@@ -24,6 +24,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ makeWrapper wrapGAppsHook ];
   buildInputs = [ gsettings-desktop-schemas ] ++ systemLibs;
+  propagatedBuildInputs = [ xdg_utils ];
```

Comment:
Can we reuse the existing wrapper?

Reply:
Is there a benefit in using the wrapper? I think this much clearer.

Reply:
I asked on IRC: 

```
18:28:00 <symphorien[m]> <supersandro2000 "propagatedBuildInputs or makeWra"> propagated build inputs only work inside a nix build (or nix-shell)
18:28:15 <symphorien[m]> so only wrapping PATH or patching the source is correct
```

Reply:
Ah, good to know. Will update the wrapper.

Reply:
Hmm... wrapping PATH does not seem to have any effect for the offending call here: https://github.com/JabRef/jabref/blob/93ad49999badd69c82b052b7d28d4df2fdf24194/src/main/java/org/jabref/gui/desktop/os/Linux.java#L36

And for patching the source we first need to have it build from source...